libs/filagui/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ endif()
 
 # Optimize materials for performance in Release mode.
 set(MATC_FLAGS -a all)
-if (CMAKE_BUILD_TYPE MATCHES Release)
+if (NOT CMAKE_BUILD_TYPE MATCHES Release)
     set(MATC_FLAGS ${MATC_FLAGS} -O)
```

Comment:
Did you mean to change the `-O` to `-g` here?

Reply:
Very much so, good catch